### DISCUSSION TITLE
PHP Fatal error:  Uncaught UnexpectedValueException: Value 'user_initiated'

### DISCUSSION THREAD

**Discussion by derrickobedgiu1** [netflie/whatsapp-cloud-api#76] [Ideas]
The new version is having a typo somewhere:
PHP Fatal error:  Uncaught UnexpectedValueException: Value 'user_initiated' is not part of the enum Netflie\WhatsAppCloudApi\WebHook\Notification\Support\ConversationType
Apparently instead of user_initiated, it's having customer_initiated
PATH: Netflie\WhatsAppCloudApi\WebHook\Notification\Support\ConversationType

**Comment by aalbarca** [comment]
Hi @derrickobedgiu1 

Thanks so much to report the error. Can you please open a issue?

BTW WhatsApp documentation says that the correct value is `customer_initiated`: https://developers.facebook.com/docs/whatsapp/cloud-api/webhooks/components#statuses-object

Can you provide the request received?

Thanks.
